recipes/gtabview/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "gtabview" %}
+{% set version = "0.10" %}
+{% set sha256 = "f791b0a103c9eeb9aa7c76c47c85f16d682d77c07a6f433a0d8a50bd9e7df7e1" %}
```

Comment:
If you [use grayskull](https://github.com/conda-incubator/grayskull) to create recipes from PyPI you won't get deprecated syntax like the unnecessary jinja variable for the checksum or the quotes in the build script.

Reply:
I removed the extra quotes, but since I'm not planning to use greyskull at this time I kind of like having the relevant/changing parts at the beginning.